velox/functions/prestosql/aggregates/tests/MapUnionSumTest.cpp

```diff
@@ -205,7 +264,26 @@ TEST_F(MapUnionSumTest, bigintOverflow) {
 
   VELOX_ASSERT_THROW(
       AssertQueryBuilder(plan).copyResults(pool()),
-      "integer overflow: 9223372036854775797 + 30");
+      fmt::format(
```

Comment:
fmt::format is redundant; ditto other places

Reply:
Imported namespace fmt. All lines now read `format(...)` instead of `fmt::format(...)`.